client/connection/openvpn.go

```diff
@@ -35,7 +35,12 @@ func ConfigureVpnClientFactory(
 
 		statsSaver := bytescount.NewSessionStatsSaver(statsKeeper)
 		statsSender := bytescount.NewSessionStatsSender(mysteriumAPIClient, vpnSession.ID, signer)
-		statsHandler := bytescount.NewCompositeStatsHandler(statsSaver, statsSender)
+		asyncStatsSender := func(stats bytescount.SessionStats) error {
+			go statsSender(stats)
+			return nil
```

Comment:
function says should return an error, but return only nil.

Reply:
Stats sender is started in separate goroutine, to avoid openvpn notifications processing blocking. There is nothing to return